lizmap/modules/admin/controllers/logs.classic.php

```diff
@@ -168,22 +168,36 @@ public function export()
             $columns = array_keys($fetchArray);
         }
 
-        $rep = $this->getResponse('binary');
-        $rep->mimeType = 'text/csv';
-        $rep->addHttpHeader('charset', 'UTF-8');
-        $rep->doDownload = true;
-        $rep->fileName = 'lizmap_logs.csv';
-
-        $data = array();
-        $data[] = '"'.implode('";"', $columns).'"';
+        # Create temp file
+        $tempPath = jApp::tempPath('export');
+        jFile::createDir($tempPath);
+        $fileName = tempnam($tempPath, 'lizmap_logs-');
+
+        # Opening file
+        $fp = fopen($fileName, 'w');
+        # Adding encode utf8 to the file
+        fprintf($fp, chr(0xEF).chr(0xBB).chr(0xBF));
+        # Adding first CSV line
+        fputcsv($fp, $columns);
```

Comment:
Did you need a specific delimiter for the csv file?
If yes, you can add it when you use `fputcsv`
eg. Use `;` as delimiter -> `fputcsv($fp, $columns, ';');`

Reply:
I don't need a specific delimiter. I copy your code.

@mdouchin @nboisteault @drillinP do you think it is necessary to use a specific delimiter ?